lib/server.js

```diff
@@ -8,22 +8,27 @@ module.exports = {
 
   createServer (config) {
     const server = new Hapi.Server()
-    server.connection({
+    const defaultOptions = {
       host: config.web.host,
       port: config.web.port,
       routes: {
         files: {
           relativeTo: config.main.paths.root
         }
       }
-    })
+    }
+
+    server.connection(_.defaultsDeep(config.web.options || {}, defaultOptions))
```

Comment:
Not really. It will allow to add custom options to Hapi server. For example I need to add 
```
{
  routes: {
    cors: true
  }
}
```